client/fingerprint/consul.go

```diff
@@ -12,16 +12,21 @@ import (
 	"github.com/hashicorp/nomad/nomad/structs"
 )
 
+const (
+	available   = "available"
```

Comment:
These should be consulAvailable/consulUnavailable as they are constants available throughout the fingerprint package.